client/verta/tests/conftest.py

```diff
@@ -277,6 +289,17 @@ def client(host, port, email, dev_key):
     print("[TEST LOG] test teardown completed {} UTC".format(datetime.datetime.utcnow()))
 
 
+@pytest.fixture
+def client_2(host, port, email_2, dev_key_2):
+    # For collaboration tests
+    print("[TEST LOG] test setup begun {} UTC".format(datetime.datetime.utcnow()))
```

Comment:
Let's have this fixture skip its tests if the credentials aren't present.
```suggestion
    if not (email_2 and dev_key_2):
        pytest.skip("second account credentials not present")
    print("[TEST LOG] test setup begun {} UTC".format(datetime.datetime.utcnow()))
```
Wow this diff is terrible

Reply:
Added.